spec/scheduledsubmission.mdown

```diff
@@ -0,0 +1,105 @@
+# JMAP Extension for Scheduled EmailSubmission
+
+# Capabilities
+
+Servers supporting this specification MUST add a property named
+"urn:ietf:params:jmap:scheduledsubmission" to the capabilities object.  The
+value of this property is an empty object.
+
+# EmailSubmission
+
+EmailSubmission objects have the following additional properties:
+
+- **sendDelay**: `UnsignedInt`
+  An additional time (in seconds) that the server will hold the email before
+  actually sending it. A client may use a standard EmailSubmission/set call to
+  cancel the email submission before this time passes.
+
+  This property may be specified during creation, and it is never returned by
+  the server.  This property exists to allow the client to specify a minimum
+  delay to be computed by the server, eliminating complications of clock skew
+  or transit time.
+- **sendAt**: `UTCDate`
+  When this extension is in effect, this property, defined in RFC 8621, may be
+  set by the client on creation.  It remains immutable.  The Date header on the
+  scheduled email should match the sendAt time, so rescheduling should be
+  performed by canceling the old EmailSubmission, and scheduling a replacement
+  email.
+- **onSend**: `SendAction`
+  An action to take when the submission has been successfully sent.
+
+  A *SendAction* object has the following properties:
+
+  - **addToMailboxId**: `Id`
+  A mailboxId to add to the Email associated with the EmailSubmission when it
+  has been successfully submitted.
```

Comment:
Why not a generic `EmailSet/update` input? 

This might allow for moving the email and set some keywords for instance.